<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MuQT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="MainNotebook">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="ChatRoomsLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Chat Rooms</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="ChatRoomsLayout">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="PrivateChatsLabel">
        <property name="accessibleName">
         <string/>
        </property>
        <attribute name="title">
         <string>Private Chat</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="PrivateChatsLayout"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TransfersLabel">
        <attribute name="title">
         <string>Transfers</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Uploads</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeWidget" name="Uploads">
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget_2">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Downloads</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeWidget" name="Downloads">
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="SearchLabel">
        <attribute name="title">
         <string>Search</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="UserinfoLabel">
        <attribute name="title">
         <string>User Info</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="BuddylistLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Buddy list</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="InterestsLabel">
        <attribute name="title">
         <string>Interests</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="UserbrowseLabel">
        <attribute name="title">
         <string>User browse</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="logWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect_to_daemon"/>
    <addaction name="actionDisconnect_from_daemon"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Away"/>
    <addaction name="actionCheck_my_privileges"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuModes">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Modes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuScripts">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Scripts</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCommands"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Museeq"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuModes"/>
   <addaction name="menuScripts"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect_to_daemon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect to daemon</string>
   </property>
  </action>
  <action name="actionDisconnect_from_daemon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disconnect from daemon</string>
   </property>
  </action>
  <action name="actionToggle_Away">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle away</string>
   </property>
  </action>
  <action name="actionCheck_my_privileges">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Check my privileges</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCommands">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Commands</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Museeq">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About Muqt</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
